addition/add.cpp

```diff
@@ -0,0 +1,10 @@
+#include<iostream>
```

Comment:
can you move this file inside a directory named basic_mathematics

Reply:
@t2013anurag Sure!

Reply:
@t2013anurag Should I create a new directory?

Reply:
https://github.com/hacktoberfest17/programming/tree/master/mathematical_operations

I just added a commit which created this directory, move the file in this.